03-feedback.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: Instructor Training
+subtitle: Feedback Strategy
+minutes: 180
+---
+> ## Learning Objectives {.objectives}
+>
+> * FIXME
+
+##Introduction
+
+In our discussion of our three-stage model of expertise, we concluded that most scientists don't need to be expert software developers; it's enough for them to be able to solve a recurring cast of coding problems in more-or-less familiar circumstances, which is the halmark of a competent practitioner. Teaching, however, is in some sense always a novel problem. Students will always come up with new ways to misunderstand things, and so the flexibility afforded by mastery in teaching is always valuable. How does one go about becoming a master instructor?
+
+The popular ten thousand hours number is certainly a myth. Many people have driven for more than ten thousand hours, and are little closer to mastery than after their driver's exam. Something more than mere repetition is required.
+
+Positing additional linkages between ideas is necessary but also not sufficient; there needs to be some mechanism to scrutinize and validate those new links and the strategies that come out of them.
+
+Musicians, athletes, actors and politicians have been doing it for years - the key is simple feedback. By having a teacher or even a colleague observe and give constructive criticism that is specific and actionable, we can iterate on our practice and get that invaluable second perspective on whether what we're doing is working, and where it's falling short. In time, with enough criticism, we can begin to usefully criticize our own practice, and this self-reflective criticism greatly accelerates the process of gaining mastery. But to begin with, getting feedback from our colleagues starts the process.
```

Comment:
On Mon, Apr 06, 2015 at 11:20:43AM -0700, Bill Mills wrote:
> I think the idea is that that useful self-reflection is actually a
> high-level skill, and it takes time to get there. But, I'd love to
> see a re-write that accommodates both points while still leading
> smoothly into talking about (external) feedback.

How about something like:

  Mastering a skill is a process of practice and feedback.  If you
  don't test yourself or judge what's working and what's not working,
  you won't know what to improve.  It's possible to critique your own
  performance, but it takes time to collect the experience necessary
  to make informed decisions in a new arena.  It's much more
  efficient, especially when you're just getting started, to get
  feedback from a more experienced coach or colleague.